src/helpers/substrate.ts

```diff
@@ -1,5 +1,22 @@
 import { EventRecord } from '@polkadot/types/interfaces';
 import { ApiPromise } from '@polkadot/api';
+import Big from 'big.js';
+import { PalletBalancesAccountData } from '@polkadot/types/lookup';
+
+/// This function is used to get the frozen account balance from the account data.
+/// Some types define the frozen balance as `frozen` and others as `miscFrozen` and `feeFrozen`.
```

Comment:
So this is coming from `pallet_balances` types in polkadot. Once you updated the `@polkadot/types` dependency, shouldn't it only be one case?

Or is this such that the portal is compatible with the not-yet-updated versions of the runtime?

Reply:
I guess it is to make this code useful for all three chains as long as the 1.1.0 is not yet fully rolled out.